packages/mui-joy/src/Chip/Chip.tsx

```diff
@@ -63,21 +63,21 @@ const ChipRoot = styled('div', {
         '--Chip-decoratorChildHeight':
           'calc(min(1.125rem, var(--_Chip-minHeight)) - 2 * var(--variant-borderWidth, 0px))',
         '--Icon-fontSize': theme.vars.fontSize.sm,
-        '--_Chip-minHeight': 'var(--Chip-minHeight, 1.5rem)',
+        '--_Chip-minHeight': 'var(--Chip-minHeight, 1.25rem)',
         gap: '0.25rem',
       }),
       ...(ownerState.size === 'md' && {
         '--Chip-paddingInline': '0.75rem',
```

Comment:
@siriwatknp I'm having a hard time trying to tweak the padding when there are icons on the chip, can you help me with this? I think it's on `--Chip-decoratorChildOffset`, but it's complex to understand how to adjust it.

This is what happened when I changed the chip height:
<img width="240" alt="Screenshot 2023-08-23 at 15 43 14" src="https://github.com/mui/material-ui/assets/37222944/db8916ff-11ea-42fc-bb8b-79292cdc54b3">

Reply:
I will take a look.

Reply:
Done!